tests/run/pure_fused.py

```diff
@@ -0,0 +1,39 @@
+# mode: run
+# tag: fused,
+# tag: pure3.7
+
+#cython: annotation_typing=True
+
+from __future__ import annotations
+
+import cython
+
+class TestCls:
+    def func1(self, arg: NotInPy):
```

Comment:
Wouldn't a string type name make this available in older Py3 versions?

Reply:
A string type name didn't work. It was what I originally tried. I'll try to see if I can work out why before revising this though.